app/react/Metadata/actions/metadataExtractionActions.ts

```diff
@@ -8,6 +8,20 @@ import { t } from 'app/I18N';
 import { RequestParams } from 'app/utils/RequestParams';
 import { ClientFile } from 'app/istore';
 
+const getAndUpdateCoercedValue = async (params: RequestParams, model: string) => {
+  const { value: coercedValue, success } = await entitiesAPI.coerceValue(params);
+  if (!success) {
+    return [
+      notificationActions.notify(
+        t('System', 'Value cannot be transformed to the correct type', null, false),
+        'danger'
+      ),
+      formActions.change(model, 0),
```

Comment:
I'm not sure if 0 applies for every case where success is false, a date for example, is it possible to have this value also as a param?

Reply:
Fixed